formbricks/formbricks
# chore: Add Table of Contents to README

## What does this PR do?



This PR adds a Table of Contents (TOC) to the README of the Formbricks project. The TOC provides quick navigation links to different sections of the README, improving the overall accessibility and usability of the document. This change enhances the user experience and makes it easier for readers to find relevant information.




## Type of change




## How should this be tested?



- Open the README file.
- Verify that a Table of Contents section is present with clickable links to different sections of the document.

## Checklist



### Required


### Appreciated
